### ISSUE TITLE
error:not such file or directory:  test_cls.gt.jsonl 

### ISSUE THREAD

**Issue by loralyc** [yuantailing/ctw-baseline#16]
i am sorry to disturb you again,
i want to ask whether the ground truth of testing set is public?
and can we get the visualization results about the classification and detection?

**Comment by yuantailing** [comment]
No, you can only access ground truth through evaluation server (only metrics, no visualization results).

**Comment by xuqingquan789** [comment]
@yuantailing How can I know  the highest classification accuracy it can be  for 1000 categories characters without testing set ground truth?

**Comment by yuantailing** [comment]
Train to convergence, then submit to evaluation server.

Do you mean the accuracy for top-1000 categories, not for all categories? We don't think it's a reasonable metric so it is not provided.
